grafana-plugin/src/containers/ChannelFilterForm/ChannelFilterForm.tsx

```diff
@@ -101,6 +101,7 @@ const ChannelFilterForm = observer((props: ChannelFilterFormProps) => {
       {!data?.is_default && (
         <IncidentMatcher
           regexp={filteringTerm}
+          alert_receive_channel_id={alertReceiveChannelId}
```

Comment:
_nitpick_ I believe React props tend to be camel cased vs snake case

Reply:
🤦 thank you